depends/README

```diff
@@ -0,0 +1,55 @@
+This is a system of building and caching dependencies necessary for building
+Bitcoin.
```

Comment:
Building Armory, you mean? ;) Initial commits are fine but there should eventually be some sort of statement in at least some of these files stating that this is for Armory, and that the system was lifted from BC Core. Copyright statements for Armory and BC Core may also be desirable in some instances. (In general, if there's a copyright notice for BC Core and we add/mod something, there should be an Armory copyright alongside the BC Core copyright.)